src/WebJobs.Script/Host/FunctionMetadataManager.cs

```diff
@@ -249,6 +252,11 @@ private void AddMetadataFromCustomProviders(IEnumerable<IFunctionProvider> funct
 
             _logger.FunctionsReturnedByProvider(totalFunctionsCount, MetadataProviderName);
 
+            if (totalFunctionsCount == 0)
```

Comment:
Also, should we do this inside `LoadFunctionMetadata` instead of `AddMetadataFromCustomProviders` method?